packages/cli/src/bin/pipcook.ts

```diff
@@ -46,7 +46,7 @@ import devPlugin from '../actions/dev-plugin';
   program
     .command('init')
     .option('-c, --client <string>', 'specify your npm client.')
-    .option('-b, --beta', 'use or update the beta version')
```

Comment:
We can't deprecate the option directly, it's good to bypass it the version.

Reply:
Updated.